src/index.html

```diff
@@ -6,7 +6,18 @@
   </head>
 
   <body>
-    <div class="grid">
+    <div class="container">
+      <h1 class="login-header">Overlook</h1>
+      <div class="login-view">
+        <div class="credentials">
+          <span>Username: </span><input type="text" id="username"></input>
+          <span>Password: </span><input type="text" id="password"></input>
+          <button id="login">Log in</button>
+      </div>
+      </div>
+    </div>
+
+    <div id="user-dashboard" class="grid hidden">
```

Comment:
You might change these later but changing the divs to section/article might lean more towards semantic